app/models/ability.rb

```diff
@@ -0,0 +1,18 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize user
+    if user.admin?
+      can :manage, :all
+      cannot :create, Borrowing
+      cannot :read, BorrowItem
+      cannot :create, BorrowItem
+      cannot :destroy, BorrowItem
+    else
+      can :create, BorrowItem
+      can :create, Borrowing
+      can :read, Borrowing
```

Comment:
nó phải là kiểu 
```suggestion
      can :read, Borrowing, user_id: user.id 
```
chứ nhỉ